src/components/VideoCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+import './VideoCard.scss'
+
+const VideoCard = ({ videoLink, header, content, className }) => {
+  return (
+  <div className={`card-container ${className}`}>
+    <div className="video-card">
+      <div className="video-card__video-container">
+        <iframe 
+          className="video-card__video"
+          title="Preview Concert Video"
+          width="560" 
+          height="315"
+          src={`${videoLink}`} 
```

Comment:
This is pretty redundant, here is a more concise way: 

```suggestion
          src={videoLink} 
```